docs/release.md

```diff
@@ -75,35 +75,54 @@ function of the appropriate pallets.
+2. **The WebSocket url of the reference node**:
+	- Statemint: `wss://statemint-rpc.polkadot.io`
+    - Statemine: `wss://statemine-rpc.polkadot.io`
+    - Westmint: `wss://westmint-rpc.polkadot.io`
+3. **A url to a Linux binary for the node containing the runtime to test**: Paste the URL of the latest release-candidate binary from the draft-release on Github. The binary has to previously be uploaded to S3 (Github url link to the binary is constantly changing)
+    - E.g: https://releases.parity.io/cumulus/v0.9.270-rc3/polkadot-parachain
+4. **The name of the chain under test. Usually, you would pass a local chain**:
+	- Statemint: `statemint-local`
+    - Statemine: `statemine-local`
+    - Westmint: `westmint-local`
+5. Click **Run workflow**
+
+When the workflow is done, click on it and download the zip artifact, inside you'll find an `output.txt` file. The things to look for in the output are lines like:
 
 - `[Identity] idx 28 -> 25 (calls 15)` - indicates the index for Identity has changed
-- `[+] Society, Recovery` - indicates the new version includes 2 additional modules/pallets
+- `[+] Society, Recovery` - indicates the new version includes 2 additional modules/pallets.
 - If no indices have changed, every modules line should look something like `[Identity] idx 25 (calls 15)`
 
-**Note**: Adding new functions to the runtime does not constitute a breaking change as long as the indexes did not
-change.
+**Note**: Adding new functions to the runtime does not constitute a breaking change
+as long as the indexes did not change.
 
-**Note**: Extrinsic function signatures changes (adding/removing & ordering arguments) are not caught by the job, so
-those changes should be reviewed "manually"
+**Note**: Extrinsic function signatures changes (adding/removing & ordering arguments) are not caught by the job, so those changes should be reviewed "manually"
 
 ### Benchmarks
 
-The Benchmarks can now be started from the CI. First find the CI pipeline from
-[here](https://gitlab.parity.io/parity/mirrors/cumulus/-/pipelines?page=1&scope=all&ref=release-parachains-v9220) and
-pick the latest.
+The Benchmarks can now be started from the CI. First find the CI pipeline from [here](https://gitlab.parity.io/parity/mirrors/cumulus/-/pipelines?page=1&scope=all&ref=release-parachains-v9220) and pick the latest.
+[Guide](https://github.com/paritytech/ci_cd/wiki/Benchmarks:-cumulus)
+
+### Integration Tests
+
+Until https://github.com/paritytech/ci_cd/issues/499 is done, tests will have to be run manually.
+1. Go to https://github.com/paritytech/parachains-integration-tests and checkout to the release branch.
+E.g. https://github.com/paritytech/parachains-integration-tests/tree/release-v9270-v0.9.27
+for `release-parachains-v0.9.270`
+2. Clone `release-parachains-<version>` branch from Cumulus
+3. `cargo build --release`
+4. Copy `./target/polkadot-parachain` to `./bin`
+5. Clone `it/release-<version>-fast-sudo` from Polkadot
```

Comment:
what's the `it` dir?

There's not always a fast sudo branch set up (it's a manual process) so better to cherry pick https://github.com/paritytech/polkadot/commit/791c8b8f9e6fe00b631f2cace0d6d65151487eb4 and run `find . -type f -name "*.toml" -print0 | xargs -0 sed -i '' -e 's/polkadot-vX.X.X/polkadot-v<version>/g'`

Reply:
`it/` is just coming from _integration-tests_. That is a branch that I always prepare for each release. It was previously `nacho/`

Reply:
But ok, better to show how to prepare it